Update databases.md (#40)
moj-analytical-services/user-guidance: source/documentation/data-docs/curated-databases-docs/databases.md

# Databases

The data engineering team curate and maintain databases on the analytical platform that can be made accessible to users. All of our databases are currently deployed and accessible as Athena Databases. We currently support users being able to query these databases via the [Athena SQL workbench](#amazon-athena), [R and Python](#dbtools).

> To make a data access request to these databases or see what databases we curate please go to [this repository](https://github.com/moj-analytical-services/data-engineering-infrastructure) and read the readme for more details. 

Reviewer: ```suggestion
To see what databases are available and how to request access, see the [README](https://github.com/moj-analytical-services/data-engineering-infrastructure/blob/master/README.md) in the [data-engineering-infrastructure](https://github.com/moj-analytical-services/data-engineering-infrastructure) repository on GitHub.
```

> >N.B. The repository is private so you will have to be a member of this (moj-analytical-services) github organisation to see the repo.

Reviewer: ```suggestion
> Note, you must be a member of the [moj-analytical-services](https://github.com/moj-analytical-services) GitHub organisation to access the repository.
```
